documentation/asciidoc/computers/config_txt/conditional.adoc

```diff
@@ -32,13 +32,13 @@ The conditional model filters apply according to the following table.
 | Model 4B, Pi 400, Compute Module 4, Compute Module 4S
 
 | `[pi5]`
-| Raspberry Pi 5, Compute Module 5, Pi 500
+| Raspberry Pi 5, Compute Module 5, Pi 500, Pi 500+
```

Comment:
Super minor nitpick, but for consistency with the other entries in this table, maybe it's worth moving CM5 to the end of the list?